template-only-docs/set-up-dependency-management.md

```diff
@@ -1,6 +1,6 @@
 # Dependency Management with Renovate
 
-Out of the box this repo uses [Renovate](https://docs.renovatebot.com/) for dependency management. More information on the decision to try renovate can be found [here](https://github.com/navapbc/template-application-nextjs/blob/main/docs/decisions/0002-use-renovate-for-dependency-updates.md). Renovate is free and open-source and allows us to bundle dependency updates together and customize their scheduling.
+Out of the box this repo uses [Renovate](https://docs.renovatebot.com/) for dependency management. More information on the decision to try renovate can be found [here](../docs/decisions/template/0001-use-renovate-for-dependency-updates.md). Renovate is free and open-source and allows us to bundle dependency updates together and customize their scheduling.
```

Comment:
Should I now strip out renovate now that dependabot can bundle 🤔 Can file a follow-up issue

Reply:
I don't feel strongly! It's working and its [Dependency Dashboard](https://github.com/navapbc/template-application-nextjs/issues/47) is nice, so I don't think there's any urgency at the moment at least.